app/summary.py

```diff
@@ -0,0 +1,50 @@
+import pickle
+import pandas as pd
+from datetime import datetime
+
+from jjit import search_jjit
+from pracuj import search_pracuj
+
+db_path = 'offers_db'
+
+def prepare_tech_dict():
+
+    offers_db = pickle.load(open(db_path, 'rb'))
```

Comment:
Niepotrzebnie pusta linia po sygnaturze funkcji, za to przed nią powinny być dwie

Reply:
Co do wczytywania pickle'a to raczej zaleca się wersję z context managerem, czyli "with open...". Nie pozostawia pliku otwartym